system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -40,6 +40,17 @@ function countTopSitesIconsTypes(topSites) {
 }
 
 export class _TopSites extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = this.DEFAULT_STATE = {
```

Comment:
This multi assign is disallowed by eslint now. See this for an example of how to fix:
https://github.com/mozilla/activity-stream/pull/3948/commits/1f8682475fa4a3fc3c12f597e172e496598db805#diff-8a848a318e8cc982dc70dfd24807f0ef